packages/budget/src/components/Budget101/index.js

```diff
@@ -1,15 +1,14 @@
 import React from "react";
-import StoryCard from "../StoryCard/StoryCard";
+import { CivicStoryCard } from "@hackoregon/component-library";
 import PieChart from "./PieChart";
 import DataTable from "./DataTable";
-import { BUDGET_101 } from "../constants";
 
 import "./Budget101.css";
 import "./stub.css";
 
 function Budget101() {
   return (
-    <StoryCard title="Budget 101" collectionId="budget" cardId={BUDGET_101}>
+    <CivicStoryCard watermark={<div />} footer={false} title="Budget 101">
```

Comment:
This totally works, but I could see this `<div />` being confusing down the road. Would it make more sense to instead add an optional `hideWatermark` prop to CivicStoryCard?

Reply:
I thought about this, and thought that leaving the `watermark` prop was more flexible without adding additional props to CivicStoryCard